### ISSUE TITLE
Topic change is not displayed in Slack

### ISSUE THREAD

**Issue by joaopapereira** [AgileVentures/HangoutConnection#31]
When we start the hangout and change the topic that name is not passed to Slack.

Instead of using:
startData.title

As the title we should use
gapi.hangout.getTopic()

**Comment by tansaku** [comment]
yeah, this has been an annoying thing for a while - would love to get it fixed - thanks for flagging @joaopapereira

**Comment by tansaku** [comment]
@joaopapereira we tested this manually on staging and it does not seem to update the title.
  It looks like gapi.hangout is nil:

![gapi.hangout nil](https://www.dropbox.com/s/i7xos3ixj1gmcpr/Screenshot%202016-10-17%2016.00.42.png?dl=1)

**Comment by tansaku** [comment]
I think @YaroSpace looked into this back in the day, and it make not work with the API:

https://code.google.com/p/google-plus-platform/issues/detail?id=956
